masonry/src/properties/box_shadow.rs

```diff
@@ -4,10 +4,12 @@
 use std::any::TypeId;
 
 use vello::Scene;
-use vello::kurbo::{Affine, Insets, Point, RoundedRect};
+use vello::kurbo::{Affine, BezPath, Insets, Point, RoundedRect, Shape as _, Size};
+use vello::peniko::Mix;
 use vello::peniko::color::{AlphaColor, Srgb};
 
 use crate::core::{Property, UpdateCtx};
+use crate::properties::CornerRadius;
 
 // TODO - This is a first implementation of box shadows. A full version would need
 // to address the following points:
```

Comment:
Is this full section still correct? The first item at least looks no longer true.